packages/broker/src/plugins/brubeckMiner/config.schema.json

```diff
@@ -25,7 +36,7 @@
                 "string",
                 "null"
             ],
-            "default": "stun.streamr.network:5349"
+            "default": "stun.sipgate.net"
```

Comment:
I'm assuming this is the one used by the nat-type detection plugin?

Reply:
Yes. `streamr.network:5349` doesn't support that yet.

Reply:
Slight correction, the library uses an out-dated version of STUN and `stun.sipgate.net` still runs using that version. We could implement our own NAT-type detection algorithm for the newer STUN versions.